beta-prompts/beta_prompts/templating_framework.py

```
@@ -0,0 +1,1029 @@
+                    "description": metadata.description,
+                    "quality_score": metadata.quality_score,
+                    "task_types": metadata.task_types,
+                    "domains": metadata.domains,
+                    "complexity_levels": metadata.complexity_levels,
+                    "tags": metadata.tags,
+                }
+            )
+
+        return results
+
+    def export_template(self, template_id: str, format: str = "json") -> str:
```

Ruff (0.12.2)

Function argument `format` is shadowing a Python builtin

(A002)